web/src/main/resources/openmrs-servlet.xml

```diff
@@ -91,7 +91,8 @@
  	<bean class="org.springframework.web.servlet.handler.SimpleMappingExceptionResolver"> 
  		<property name="exceptionMappings"> 
  			<map> 
- 				<entry key="org.openmrs.api.APIException" value="uncaughtException"/> 
+ 				<entry key="org.openmrs.api.APIException" value="uncaughtException"/>
+				<entry key="org.springframework.web.method.annotation.MethodArgumentTypeMismatchException" value="uncaughtException"/>
```

Comment:
What do we do for other exceptions which are not `MethodArgumentTypeMismatchException`?